files/fr/web/html/link_types/noopener/index.html

```diff
@@ -0,0 +1,24 @@
+---
+title: 'Types de liens : noopener'
+slug: Web/HTML/Link_types/noopener
+browser-compat: html.elements.a.rel.noopener
+translation_of: 'Web/HTML/Link_types/noopener'
+---
+<p>Le mot-clé <strong><code>noopener</code></strong> de l'attribut <code><a href="/fr/docs/Web/HTML/Attributes/rel">rel</a></code> des éléments {{HTMLElement("a")}}, {{HTMLElement("area")}} et {{HTMLElement("form")}} indique au navigateur d'ouvrir la ressource cible sans donner d'accès au document précédent au nouveau contexte de navigation qui sera créé, en n'indiquant pas la propriété {{DOMxRef("Window.opener")}} sur la nouvelle fenêtre (il retourne <code>null</code>).</p>
+
+<p>Ceci est tout spécialement utile lors de l'ouverture de liens non vérifiés/fiables, afin de s'assurer qu'ils ne peuvent pas trafiquer le document d'origine via la propriété {{DOMxRef("Window.opener")}} (pour plus de détails, voir <a href="https://mathiasbynens.github.io/rel-noopener/">cet article en anglais concernant <code>rel=noopener</code></a>), lorsque l'en-tête HTTP <code>Referer</code> est fourni (à moins que <code>noreferrer</code> soit également utilisé).</p>
+
+<p>Veuillez noter que lorsque <code>noopener</code> est utilisé, les noms de cibles non vides et différents de <code>_top</code>, <code>_self</code> et <code>_parent</code> sont tous traités en tant que <code>_blank</code> pour déterminer si le lien doit s'ouvrir ou non dans un nouvel onglet/fenêtre.</p>
+
+<div class="notecard note">
+  <h4>Note</h4>
```

Comment:
Preparing the field for Markdown conversion: could I ask to use a `<p><strong>Note :</strong> Blabla...</p>` for notes?

Reply:
Ok ! Does the implementation proposed in https://github.com/mdn/translated-content/pull/1798/commits/0f0ad927b7f2548e2d12ccaabc29dff40c652eac work for you?